apps/pushes/views/diff.py

```diff
@@ -278,6 +232,15 @@ def content(self, path, rev):
         content = self._universal_newlines(content)
         return content
 
+    def parse(self, parser, content, data):
```

Comment:
Would it make sense to return an `OrderedDict` from this method? You could rename it to `parse_ordered` and use it like so:

```py
new_data.update(self.parse_ordered(parser, content))
```